robottelo/common/helpers.py

```diff
@@ -151,6 +151,32 @@ def generate_strings_list(len1=None):
     ]
 
 
+def generate_strings_dict(len1=None):
```

Comment:
This method is a near duplicate of `generate_strings_list`. Can you update `generate_strings_list` to look something like this?

    def generate_strings_list(...):
        return generate_strings_dict(...).values()